### DISCUSSION TITLE
Error in recognized wind.ww3 file in ww3_shel.

### DISCUSSION THREAD

**Discussion by AlexCrls** [NOAA-EMC/WW3#1300] [Q&A]
Hi
I am running the ww3_shel program and encountering an issue where the wind input file, wind.ww3, is not being recognized (I created with ww3_prnc.inp). The file is located in the correct directory and appears to be in the proper format, but the program does not detect it during execution.

Could you please advise on possible reasons why the wind.ww3 file might not be recognized by ww3_shel? I would also appreciate any guidance on aspects of the file or the input process that might be causing this issue.

Thank you in advance for your assistance.

Best regards,

**Comment by mickaelaccensi** [comment]
Dear Alex,

I think your issue comes from the fact that your ww3_shel.inp is not well configured. Please double check that the wind forcing is activated
Otherwise you can provide your input files and log files.

Best,

Mickael

**Reply by AlexCrls** [reply]
Hi Mickael

I am attaching the .inp file and the log file generated from its execution. Additionally, when running the program, an error occurs related to the conversion of the shel file, which I am also attaching

Thanks

Best

[slurm-168850.txt](https://github.com/user-attachments/files/16980758/slurm-168850.txt)
[ww3_shel.inp.txt](https://github.com/user-attachments/files/16980760/ww3_shel.inp.txt)
[ww3_shel.txt](https://github.com/user-attachments/files/16980761/ww3_shel.txt)


Alex

**Comment by mickaelaccensi** [comment]
ok the issue is due to the ice forcing which are not used depending on the parameterization you have chosen in the switch file.
Please use the ww3_shel.nml (in model/nml) instead of the ww3_shel.inp to easily solve this issue.

**Comment by AlexCrls** [comment]
Hi
Thank you very much for your response. 

I already ran it with the .nml file, and the model executed successfully. If I modify the parameterizations in the ww3_grid.inp, could this solve the error in the shel.inp?

Regards

**Comment by mickaelaccensi** [comment]
you can either activate the ice parameterization in the switch file (IC2 IS2) and rerun the ww3_grid to produce a new mod_def.ww3 or remove the "ice" lines in the ww3_shel.inp but I highly recommend you to use the nml files instead of the inp files since nml files will be the default config files (inp files should be removed within 1 or 2 years possibly) and nml files offers more features than inp files
